### ISSUE TITLE
Sphinx error on queue.Queue

### ISSUE THREAD

**Issue by jaraco** [jaraco/jaraco.itertools#13]
Attempting to build the docs now, they fail with:

```
jaraco.itertools main $ tox -e docs
docs develop-inst-noop: /Users/jaraco/code/main/jaraco.itertools
docs installed: alabaster==0.7.12,attrs==21.2.0,Babel==2.9.1,black==21.9b0,certifi==2021.5.30,charset-normalizer==2.0.6,click==8.0.1,coverage==5.5,docutils==0.17.1,filelock==3.0.12,flake8==3.9.2,idna==3.2,imagesize==1.2.0,importlib-metadata==4.8.1,inflect==5.3.0,iniconfig==1.1.1,jaraco.context==4.0.0,jaraco.functools==3.3.0,-e git+gh://jaraco/jaraco.itertools@5fc745a678dc29e04f4962f5ac9c8b57e2683a0a#egg=jaraco.itertools,jaraco.packaging==8.2.1,Jinja2==3.0.1,MarkupSafe==2.0.1,mccabe==0.6.1,more-itertools==8.10.0,mypy==0.910,mypy-extensions==0.4.3,packaging==21.0,pathspec==0.9.0,pep517==0.11.0,platformdirs==2.3.0,pluggy==1.0.0,py==1.10.0,pycodestyle==2.7.0,pyflakes==2.3.1,Pygments==2.10.0,pyparsing==2.4.7,pytest==6.2.5,pytest-black==0.3.12,pytest-checkdocs==2.7.1,pytest-cov==2.12.1,pytest-enabler==1.2.0,pytest-flake8==1.0.7,pytest-mypy==0.8.1,python-dateutil==2.8.2,pytz==2021.1,regex==2021.8.28,requests==2.26.0,rst.linker==2.2.0,six==1.16.0,snowballstemmer==2.1.0,Sphinx==4.2.0,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==2.0.0,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.5,toml==0.10.2,tomli==1.2.1,typing-extensions==3.10.0.2,urllib3==1.26.6,zipp==3.5.0
docs run-test-pre: PYTHONHASHSEED='3206874439'
docs run-test: commands[0] | python -m sphinx -W --keep-going . /Users/jaraco/code/main/jaraco.itertools/build/html
Running Sphinx v4.2.0
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: [config changed ('project')] 2 added, 0 changed, 0 removed
reading sources... [ 50%] history
reading sources... [100%] index

looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [ 50%] history
writing output... [100%] index

/Users/jaraco/code/main/jaraco.itertools/jaraco/itertools.py:docstring of jaraco.itertools.FetchingQueue:1: WARNING: py:class reference target not found: queue.Queue
generating indices... genindex py-modindex done
writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build finished with problems, 1 warning.
ERROR: InvocationError for command /Users/jaraco/code/main/jaraco.itertools/.tox/docs/bin/python -m sphinx -W --keep-going . /Users/jaraco/code/main/jaraco.itertools/build/html (exited with code 1)
______________________________________________________________________________ summary ______________________________________________________________________________
ERROR:   docs: commands failed

```

This sounds familiar. I think something in late versions of Sphinx added links to the superclasses, creating an implicit dependency on the Python docs.

**Comment by jaraco** [comment]
Aha. It was jaraco/skeleton#36.

@webknjaz Can you propose the fix?

**Comment by webknjaz** [comment]
I have a static site showing all the linkable objects from select sites. I checked the CPython upstream docs https://webknjaz.github.io/intersphinx-untangled/docs.python.org/ for `queue.Queue` and found that it exists as a class:
```rst
:py:class:`queue.Queue`
```
This meant that it's linkable via intersphinx. Intersphinx extends the number of objects that Sphinx can reach. But I then noticed that `docs.python.org` was not added to the mapping explaining why the objects from the site weren't linkable. While adding the mapping, I realized that the extension is not even added.
So now I think that #14 should address this issue.
